webapp/src/components/mobilitymap/ReservationCard.js

```diff
@@ -37,22 +37,22 @@ const ReservationCard = ({
   if (vehicle.gotoPrivateAccount) {
     action = (
       <>
-        <p>To get started, we&rsquo;ll set up a private account for you in Lime.</p>
+        <p>{t("mobility:setup_lime_private_account_title")}</p>
```

Comment:
Oh, this should use `vehicle.vendorService.vendorName`, nothing should be lime-specific.